api/recompress_chunk.md

```diff
@@ -11,23 +11,22 @@ api:
 
 # recompress_chunk() <Tag type="community" content="Community" />
 
-Recompresses a compressed chunk that had more data inserted after compression.
+Recompresses a compressed chunk while applying new compression settings.
```

Comment:
Not correct, this is deprecated and now a wrapper for decompress_chunk(chunk,true,false)